test/basic.test.js

```diff
@@ -681,3 +681,36 @@ test('correctly log NaN', async (t) => {
   const { num } = await once(stream, 'data')
   t.is(num, null)
 })
+
+test('correctly skip function', async (t) => {
+  const stream = sink()
+  const instance = pino(stream)
+
+  const o = { num: NaN }
+  instance.info(o, () => {})
+
+  const { msg } = await once(stream, 'data')
+  t.is(msg, undefined)
+})
+
+test('correctly skip Infinity', async (t) => {
+  const stream = sink()
+  const instance = pino(stream)
+
+  const o = { num: NaN }
+  instance.info(o, Infinity)
+
+  const { msg } = await once(stream, 'data')
+  t.is(msg, null)
+})
+
+test('correctly skip Infinity', async (t) => {
+  const stream = sink()
+  const instance = pino(stream)
+
+  const o = { num: NaN }
+  instance.info(o, 42)
```

Comment:
I think the test name is wrong here-- it appears to simply be a copy/paste of the test name above. Perhaps the intent was "correctly skip Number" ?

Reply:
I'm not sure this test is really doing anything actually. @mihai1voicescu can you clarify?

Reply:
I forgot to rename the test 😢. Should test that given a number, will print the number as string in the msg.
Basically testing the switch case.